### PR TITLE
Add attribution info to metadata, if available

### PR DESCRIPTION
**Pull request by johanhelsing**

### CODE REVIEW

**Comment by johanhelsing** [comment]
I needed this for my use case. Take or leave it :)

**Comment by johanhelsing** [comment]
Example:

```
~/dev/oga oga describe --verbose 12-medieval-signs
{
    "attribution": "J. W. Bjerk (eleazzaar) -- www.jwbjerk.com/art",
    "author": "eleazzaar",
    "favorites": 33,
    "files": [
        {
            "etag": "7fd3cc-4c181f2113ac0",
            "id": "12-medieval-signs_0.zip",
            "size": 8377292
        }
    ],
    "id": "12-medieval-signs",
    "licenses": [
        "CC-BY 3.0",
        "CC-BY-SA 3.0",
        "GPL 3.0",
        "GPL 2.0"
    ],
    "tags": [
        "sign",
        "RPG",
        "Fantasy",
        "medieval",
        "Fish",
        "blacksmith",
        "magic",
        "weapon",
        "Inn",
        "shop",
        "unicorn"
    ],
    "type": "Texture"
}
```

File: oga/parsing.py [hunk 1]
```diff
@@ -86,6 +93,10 @@ def parse_asset(asset_id: str, data: bytes) -> dict:
         "favorites": favorites,
         "files": files}
 
+    if attribution != None:
+        ret["attribution"] = attribution
+
+    return ret
```
Comment (numberoverzero) [inline oga/parsing.py:99]: I have a slight preference for the parser including every field, even if it's `None`.  It's a signal that the parser looked but didn't find it, whereas a new field would be omitted entirely by an older parser.

```suggestion
        "files": files,
        "attribution": attribution}

    return ret
```

File: oga/primitives.py [hunk 2]
```diff
@@ -69,5 +71,10 @@ def to_json(self) -> dict:
             "favorites": self.favorites,
             "files": [file.to_json() for file in self.files]}
 
+        if self.attribution != None:
+            ret["attribution"] = self.attribution
```
Comment (numberoverzero) [inline oga/primitives.py:75]: same comment as above, I'd prefer `"attribution"` be included even when `None`.  stripping empty values at the CLI could be an option but the model shouldn't omit them.

**Comment by numberoverzero** [comment]
Looks great, thanks for the pull request!  I'm happy to add this functionality.

If you'd prefer I can merge this as-is and make those changes, just let me know.

File: oga/primitives.py (hunk 2 above)
Comment (johanhelsing) [reply oga/primitives.py:75]: I don't mind either way is fine by me.

**Comment by johanhelsing** [comment]
Just merge and make the edits you want to :)

**Comment by numberoverzero** [comment]
Thanks again for the contribution!

**Pull request merged**